README.md

```diff
@@ -39,6 +39,25 @@ The linters will write its data to the path configured in `xsoar.autoFindProblem
 
 If you wish to see the process running (or check why it's failing or not working), change `xsoar.linter.showOnSaveTerminal` to `true`.
 
+### Dev Containers
+
+The Cortex XSOAR extension can run content on development container.
+
+There are two options:
+
+1. Run content repository on Dev Container. This opens the repository on [demisto-sdk docker](https://github.com/demisto/dockerfiles/tree/master/docker/demisto-sdk), which contains `demisto-sdk` and content `content` basic requirements.
+   To activate, run the command `Open content in Dev Container` from the command pallette or right click in file or editor.
+2. Run an integration of script on dev container. This opens a workspace inside a container that is based upon the integration or script docker image (which is specified in their YAML file). This workspace is fully configured with `Python`, `Pylance`, `flake8`, `mypy` and `pytest`, allowing developing and debugging inside the integration environment.
+   To activate, run the command `Open integration/script in Dev Container` from the command pallette or right click in file or editor inside a specific integration or a script.
+   If running an integration which is on `python 2`, the installation of `mypy` will fail because it's not supported on `python 2`. In addition, in order to debug the tests it is required to install `Python` extension version `2022.2`:
```

Comment:
add a not that we are not planning to support python2 and it is just a workaround until it disabled.